AUTHORS.txt

```diff
@@ -39,3 +39,4 @@ Dan McDonald
 Hud Wahab
 Nicola Corna
 Robert Eckelmann
+Sam Condon
```

Comment:
You need also to add yourself in `docs/about/authors.rst`. Sorry for this duplication

Reply:
Gotcha, just added myself to that file.